src/Raven.Server/Config/Categories/AiConfiguration.cs

```diff
@@ -0,0 +1,73 @@
+using System.ComponentModel;
+using Raven.Server.Config.Attributes;
+using Raven.Server.Config.Settings;
+using Raven.Server.Documents.ETL.Providers.AI.Embeddings;
+
+namespace Raven.Server.Config.Categories;
+
+[ConfigurationCategory(ConfigurationCategoryType.Ai)]
+public sealed class AiConfiguration : ConfigurationCategory
+{
+    #region Embeddings Generation Task
+
+    [Description("Maximum number of documents processed in a single batch by the Embeddings Generation task. " +
+                 "Higher values may improve throughput but require more resources and higher limits from the AI service.")]
+    [DefaultValue(128)]
+    [ConfigurationEntry("Ai.Embeddings.Generation.Task.MaxBatchSize", ConfigurationEntryScope.ServerWideOrPerDatabase)]
+    public int? EmbeddingsGenerationTaskMaxBatchSize { get; set; }
+
+    [Description("Base delay for Embedding Generation task retries. The actual wait time between retry attempts depends on the configured FallbackModeStrategy. " +
+                 $"When using '{nameof(EmbeddingsGenerationRetryStrategy.Linear)}' strategy, the delay increases linearly (e.g., 15s, 30s, 45s). " +
+                 $"When using '{nameof(EmbeddingsGenerationRetryStrategy.Exponential)}' strategy, the delay increases exponentially with each retry attempt (e.g. 15s, 60s, 120s, 240s).")]
+    [DefaultValue(15)]
+    [TimeUnit(TimeUnit.Seconds)]
+    [ConfigurationEntry("Ai.Embeddings.Generation.Task.RetryDelayInSec", ConfigurationEntryScope.ServerWideOrPerDatabase)]
+    public TimeSetting EmbeddingsGenerationTaskRetryDelay { get; set; }
```

Comment:
I don't think that I like either that we have a strategy here, or that we rely on the user to provide this value.

Reply:
If you want we can simply it as in ETLs, there is only one config there 'MaxFallbackTimeInSec'

Reply:
Yes, I agree that would be better